src/app/app-layout/navbar/navbar.component.html

```diff
@@ -1,57 +1,87 @@
-<div class="bg-white z-0 flex w-screen flex-col lg:flex-row justify-start lg:justify-center transition-all" [class.h-screen]="(isDropdownOpen$ | async)">
-    <div class=" flex w-full justify-start h-24 sticky overflow-x-hidden max-w-6xl px-4 lg:px-0"
-      [class.h-screen]="(isDropdownOpen$ | async)"
-      [class.fixed]="(isDropdownOpen$ | async)">
-      <div class="w-full py-2">
-        <img class="h-full py-4 object-contain" src="../../../assets/default-logo.png" alt="" srcset="">
-      </div>
-      <div class="flex justify-end items-center">
-        <button *ngFor="let navbarItem of navbarScreenLinks"
-          class="mx-2 p-4 text-gray-700 text-sm font-semibold whitespace-nowrap hidden lg:inline"
-          [routerLink]="navbarItem.routerLink" routerLinkActive="border-b-4 text-indigo-800 border-indigo-800">
-          {{ navbarItem.title }}
-        </button>
-
-        <button app-action-button class="transition-all mx-4 px-8 py-4 border-2 border-indigo-800 
-                  font-semibold rounded-full text-sm 
+<div class="bg-white flex w-screen flex-col transition-all lg:flex-row lg:justify-center"
```

Comment:
While I was fixing several serious layout issues with this component, it was really hard to digest what's happening here. If something is more complex, add more comments to be clear for the reader.
This file now looks like this:

![image](https://user-images.githubusercontent.com/13491551/126704627-b40dd2d9-dba6-4de5-92e2-5d3781fb6c2c.png)